plugin/trino-kafka/src/main/java/io/trino/plugin/kafka/KafkaConfig.java

```diff
@@ -44,6 +51,7 @@
     private int messagesPerSplit = 100_000;
     private boolean timestampUpperBoundPushDownEnabled;
     private String tableDescriptionSupplier = FileTableDescriptionSupplier.NAME;
+    private File resourcePropertiesFile;
```

Comment:
it is a list of files

Reply:
Sounds good. I'll change it for a list of files.